### PR TITLE
Added 5 projects to Awesome oneAPI

### PR DESCRIPTION
**Pull request by sramkrishna**
Update README.md

add 5 projects to the awesome oneAPI repo.

### CODE REVIEW

**Comment by sramkrishna** [comment]
@FCLC can you look over and approve?

**Comment by FCLC** [comment]
ack

File: README.md [hunk 1]
```diff
@@ -54,7 +53,9 @@ https://oneapi.io/ for more information.
 ### AI - Machine Learning
 
 * [DQRM](https://anonymous.4open.science/r/Deep_Quantized_Recommendation_Model_DQRM-6B4D) - Deep Quantized Recommendation Model (DQRM) is a recommendation framework that is small, powerful in inference, and efficient to train.
+* [ort](https://github.com/pykeio/ort) - ort is an (unofficial) ONNX Runtime 1.15 wrapper for Rust based on the now inactive onnxruntime-rs. ONNX Runtime accelerates ML inference on both CPU & GPU.
```
Comment (FCLC) [inline README.md:56]: Is a project that has the option to link against the oneAPI libraries worth of considering?

File: README.md [hunk 2]
```diff
@@ -54,7 +53,9 @@ https://oneapi.io/ for more information.
...
 * [Performance and Portability Evaluation of the K-Means Algorithm on SYCL with CPU-GPU architectures](https://github.com/artecs-group/k-means) - This work uses the k-means algorithm to asses the performance portability of one of the most advanced implementations of the literature He-Vialle over different programming models (DPC++ CUDA OpenMP) and multi-vendor CPU-GPU architectures.
+* [dpcpp-svm](https://github.com/floatshadow/dpcpp-svm) - A DPC++ version fo ThunderSVM. The mission of ThunderSVM is to help users easily and efficiently apply SVMs to solve problems. ThunderSVM exploits GPU and multi-core CPUs to achieve high efficiency.
```
Comment (FCLC) [inline README.md]: typo, should be of not fo

File: README.md [hunk 3]
```diff
@@ -194,6 +198,7 @@ code to SYCL.
...
 * [Optical Flow Method](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/C%2B%2BSYCL/StructuredGrids/guided_HSOpticalflow_SYCLMigration) - (C++ based, from Intel) The HSOpticalFlow sample is a computation of per-pixel motion estimation between two consecutive image frames caused by movement of object or camera. Shows how to migrate CUDA based code to SYCL.
 * [PyPardisoProject](https://github.com/haasad/PyPardisoProject) - Pypardiso is a Python package for solving large sparse linear systems of equations using the Intel OneAPI Math Kernel Library Pardiso solver. It provides the same functionality as Scipy's spsolve but is faster in many cases.
+[suanPan](https://github.com/TLCFEM/suanPan) - abacus suanPan is a finite element method (FEM) simulation platform for applications in fields such as solid mechanics and civil/structural/seismic engineering. The name suanPan (in some places such as suffix it is also abbreviated as suPan) comes from the term Suan Pan (算盤), which is Chinese abacus.
```
Comment (FCLC) [inline README.md]: Project is called suanPan, why does it's description begin with "Abucus" if that's only to do with the naming as brought up at the end of the descriptor?

**Review by FCLC — CHANGES_REQUESTED** [review]
Question, typo, style?

File: README.md (hunk 1 above)
Comment (sramkrishna) [reply README.md:56]: Absolutely.

File: README.md (hunk 2 above)
Comment (sramkrishna) [reply README.md]: oops, thanks! I'll fix.

File: README.md (hunk 3 above)
Comment (sramkrishna) [reply README.md]: I think that might be a typo.

**Comment by sramkrishna** [comment]
@FCLC  fixed the typos.

**Pull request merged**